{"name": "Невозможно обновить данные без авторизации", "status": "passed", "parameters": [{"name": "field", "value": "'email'"}], "start": 1742964631799, "stop": 1742964632957, "uuid": "cbe7765b-763b-44d4-8f64-372df3934079", "historyId": "62a26e3d907d68b75d88a837ec80ef0f", "testCaseId": "50ab14e5bd59ba94665da529c21fdea1", "fullName": "tests.test_user.TestUser#test_update_unauthorized_user_data_fails", "labels": [{"name": "feature", "value": "Пользователь"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user"}, {"name": "subSuite", "value": "TestUser"}, {"name": "host", "value": "LAPTOP-QVE1U26H"}, {"name": "thread", "value": "15136-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user"}]}